src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,108 @@
+            + " WHERE id = ? AND is_deleted = FALSE;";
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement creteStatement = connection.prepareStatement(INSERT,
+                         Statement.RETURN_GENERATED_KEYS)) {
+            creteStatement.setString(1, manufacturer.getName());
+            creteStatement.setString(2, manufacturer.getCountry());
+            creteStatement.executeUpdate();
+            ResultSet keys = creteStatement.getGeneratedKeys();
+            while (keys.next()) {
+                Long id = keys.getObject(1, Long.class);
+                manufacturer.setId(id);
+            }
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert manufacture" + manufacturer, e);
+        }
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getStatement = connection.prepareStatement(SELECT_BY_ID)) {
+            getStatement.setLong(1,id);
+            ResultSet resultSet = getStatement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(createManufacturer(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't select manufacture from id = " + id, e);
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> listAllManufacturers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                 Statement getAllStatement = connection.createStatement()) {
+            ResultSet resultSet = getAllStatement.executeQuery(SELECT_ALL);
+            while (resultSet.next()) {
+                listAllManufacturers.add(createManufacturer(resultSet));
+            }
+            return listAllManufacturers;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't select all manufacture", e);
+        }
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement updateStatement = connection.prepareStatement(UPDATE)) {
+            updateStatement.setString(1, manufacturer.getName());
+            updateStatement.setString(2,manufacturer.getCountry());
+            updateStatement.setLong(3,manufacturer.getId());
+            updateStatement.executeUpdate();
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update manufacture:"
+                    + manufacturer.toString(), e);
```

Comment:
```suggestion
                    + manufacturer, e);
```
`toString()` will be called automatically. Please check and fix similar cases.